x-pack/plugins/cases/public/components/case_view/components/case_view_activity.test.tsx

```diff
@@ -162,19 +168,21 @@ describe('Case View Page activity tab', () => {
   beforeEach(() => {
     jest.clearAllMocks();
     appMockRender = createAppMockRenderer();
```

Comment:
not related but since we are here, this looks irrelevant no? Every test seems to re-initialize the `appMockRenderer` anyway

Reply:
Yes, but I would like to keep it in the case in the future we add a test and do not re-initialize the `appMockRenderer `.